src/components/playback/playbackmanager.js

```diff
@@ -2611,9 +2611,11 @@ export class PlaybackManager {
                 autoSetNextTracks(prevSource, mediaStreams, trackOptions, user.Configuration.RememberAudioSelections, user.Configuration.RememberSubtitleSelections);
                 if (trackOptions.DefaultAudioStreamIndex != null) {
                     options.audioStreamIndex = trackOptions.DefaultAudioStreamIndex;
+                    mediaSourceId = mediaSourceId || item.Id;
```

Comment:
From my testing, DefaultSubtitleStreamIndex and DefaultAudioStreamIndex don't always get set unless mediaSourceId is defined.

Reply:
Probably because of this? https://github.com/jellyfin/jellyfin/blob/master/Jellyfin.Api/Helpers/MediaInfoHelper.cs#L197

Reply:
> Probably because of this? https://github.com/jellyfin/jellyfin/blob/master/Jellyfin.Api/Helpers/MediaInfoHelper.cs#L197

Well, makes sense - indices are bound to the media source.

OK, so we need to specify `MediaSourceId` to get valid audio and subs.

From my tests, when playback is started from the item details page, it specifies `options.mediaSourceId` equal to `item.Id` (selected media source) for movies/shows and as an empty string for LiveTV.

When we use previous/next track, `options.mediaSourceId` is undefined.

My concern is that the current implementation is not obvious.

I tried optionally using `item.Id` as `mediaSourceId` when `item.Type !== BaseItemKind.TvChannel`, but there is also `Channel` and `LiveTvChannel`. 🤔

I also think we should skip `getMediaStreams` for LiveTV - less network requests, and because we are discarding audio and subs for LiveTV anyway, iirc.

Reply:
> I tried optionally using item.Id as mediaSourceId when item.Type !== BaseItemKind.TvChannel, but there is also Channel and LiveTvChannel. 🤔

I've updated it to only set mediaSourceId when it's not any of these types to hopefully make it clearer, but I'm not exactly sure what the difference between them is.

> I also think we should skip getMediaStreams for LiveTV - less network requests, and because we are discarding audio and subs for LiveTV anyway, iirc.

Made `getMediaStreams` resolve to an empty list when we're using Live TV.